core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -5434,6 +5434,14 @@ public String toString() {
           .setDescription("The port Alluxio FUSE web UI runs on.")
           .setScope(Scope.CLIENT)
           .build();
+  public static final PropertyKey FUSE_JNIFUSE_VERSION =
+      new Builder(Name.FUSE_JNIFUSE_VERSION)
```

Comment:
If it not the version of jnifuse, but the libfuse version, more straightforward name

Reply:
Changed to `alluxio.fuse.jnifuse.libfuse.version`